LanguageServers.md

```diff
@@ -124,6 +125,14 @@
   - Installation: `rustup component add rust-analyzer`
   - Command: `rust-analyzer`
 
+## Scala
+
+- [metals](https://github.com/scalameta/metals)
+  - Installation: (requires [coursier](https://get-coursier.io/docs/cli-installation)) command: `coursier install metals`
+  - formatting does not work
+  - `Import build` worskspace command is not supported
```

Comment:
This doesn't need to be mentioned here, workspace commands are not yet supported for any language server. I've opened a new issue (#60) to track the feature.